public/style/main.css

```diff
@@ -12,8 +12,24 @@ body {
 
 /*bs modification*/
 .btn {
-  border-radius    : 0px;
+  /*border-radius    : 0px;*/
+  border-radius    : 50px;
+  font-size: 1em;
+}
+
+.btn-info {
+  color: black;
+  background-color: yellow;
+  border-color: black;
+  border-width: 1px;
+}
+
+.btn-success {
```

Comment:
dont override an existing bootstrap button - then we cant use it elsewhere. create your own button if you want custom stylings, e.g. `btn-phil`

Just use google I found this tool in 30 seconds: http://blog.koalite.com/bbg/